UI/SidePanel.cpp

```diff
@@ -3033,8 +3033,8 @@ void SidePanel::RefreshInPreRender() {
 }
 
 void SidePanel::RefreshSystemNames() {
-    auto system = GetSystem(s_system_id);
-    if (!system)
+    auto this_system = GetSystem(s_system_id);
```

Comment:
`selected_system`

Reply:
although checking the selected system can probably just be commented out entirely, as the result doesn't seem to be used anywhere later.